.azure-pipelines.yml

```diff
@@ -0,0 +1,22 @@
+jobs:
+  - job: Windows
+    strategy:
+      matrix:
+        MSVC_2015:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+        MSVC_2015_NO_PCH:
```

Comment:
In theory PCH is to speed up builds. Why do we build both? To make sure our PCH header files and non-pch includes are correct? (Is no-pch even a used use case?)

Reply:
It's just to ensure that the non-PCH includes are correct, otherwise there are no reasons for not using the normal (PCH) build.